Bots/JavaScript/Consumers/CodeFirst/SimpleHostBot/dialogs/setupDialog.js

```diff
@@ -7,7 +7,7 @@ const { HostBot } = require('../bots/hostBot');
 
 const SETUP_DIALOG = 'SetupDialog';
 const CHOICE_PROMPT = 'ChoicePrompt';
-const WATERFALL_DIALOG = 'WaterfallDialog';
+const WATERFALL_STEPS = 'WaterfallSteps';
```

Comment:
I think that in this case we should have left WaterfallDialog (the steps in c# were for another variable, this is the ID for the actual dialog and WaterfallDialog is the correct term)